modules/jdbc/src/main/java/org/testcontainers/jdbc/ContainerDatabaseDriver.java

```diff
@@ -175,6 +176,14 @@ public synchronized Connection connect(String url, final Properties info) throws
         return results;
     }
 
+    private String getQueryStringWithoutTCParams(String queryString) {
+        return TC_PARAM_MATCHING_PATTERN.matcher(queryString)
```

Comment:
What if we just re-create URL without TC_ query parameters?
Something like https://stackoverflow.com/a/41725109